prow/cluster/needs-rebase_deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: needs-rebase
-        image: gcr.io/k8s-prow/needs-rebase:v20190301-2d35634
+        image: gcr.io/k8s-prow/needs-rebase:v20190304-b7b76c8
```

Comment:
@fejta This does not exist? Is this supposed to happen?

Reply:
No it is not "supposed" to happen but can: https://testgrid.k8s.io/sig-testing-prow#push-prow


https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-test-infra-push-prow/1475